src/armscan_env/envs/state_action.py

```diff
@@ -29,6 +30,9 @@ def to_normalized_array(
         # normalize translation to [-1, 1]: 0 -> -1, translation_bounds -> 1
         rotation = np.zeros(2)
         translation = np.zeros(2)
+        if self.translation[0] < 0 or self.translation[1] < 0:
```

Comment:
when would that happen? debug logging seems a bit weak, can we raise an error? Actions should always be valid, I feel

Reply:
it happens when the transformation is negative, and transforms the optimal action into negative octant. Reason for which it is necessary to reproject the optimal action into positive octant.
In the end it just defines a coordinate system of a plane, so projecting it still defines the same plane.